lnwallet/chainfee/estimator.go

```diff
@@ -468,17 +468,25 @@ func (b *BitcoindEstimator) fetchEstimate(confTarget uint32) (SatPerKWeight, err
 	// estimated fee rate from its sat/kb representation to sat/kw.
 	satPerKw := SatPerKVByte(satPerKB).FeePerKWeight()
 
+	// Bitcoind will not report any fee estimation if it has not enough
+	// data available hence the fee will remain zero. We return an error
+	// here to make sure that we do not use the min relay fee instead. This
+	// might be a bad estimate for the current bitcoin network fees.
+	if satPerKw == 0 {
+		return 0, fmt.Errorf("fee estimation data not available yet")
+	}
+
 	// Finally, we'll enforce our fee floor.
 	minRelayFee := b.minFeeManager.fetchMinFee()
 	if satPerKw < minRelayFee {
-		log.Debugf("Estimated fee rate of %v sat/kw is too low, "+
-			"using fee floor of %v sat/kw instead", satPerKw,
+		log.Debugf("Estimated fee rate of %v is too low, "+
```

Comment:
why update these log messages?

Reply:
because the unit in itself `SatPerKWeight` has a string method which already prints out the sat/kw